source/includes/_ais.md

```diff
@@ -173,13 +178,48 @@ Production | `https://openbanking.monzo.com/open-banking/v3.1/aisp/pots`
 Note that the fields we return as part of the response depend on whether your consent has the  `ReadAccountsBasic` 
 or `ReadAccountsDetailed` permission. In the former case, we will omit the Pot name and Image URL from the response.
 
-We'll only return open pots as part of our response.
+We'll only return open pots as part of our response. If a customer closes a pot, it won't appear in the response
+any more.
+
+## Direct Debits
+
+We've implemented version 3.1.2 of the [Open Banking Direct Debits specification](https://openbanking.atlassian.net/wiki/spaces/DZ/pages/1077805417/Direct+Debits+v3.1.2).
```

Comment:
Is it worth noting that we've only implemented one of the two endpoints in each of these?
In this case, we've implemented `GET /accounts/{AccountId}/direct-debits` but not `GET /direct-debits`, and so on for the other two.